spec/views/core_induction_programmes/modules/show.html.erb_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "core_induction_programmes/modules/show.html.erb" do
```

Comment:
This page already existed so I expected it to be covered by a cypress test. We're also testing a few cases rather than a single happy path which makes it slows at integration level. We can try pairing but feel it will be more painful than it looks.

Reply:
This spec has now gone in favour of a cypress test despite the fact i think it offers less coverage and less benefit